<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSerial</class>
 <widget class="QWidget" name="WidgetSerial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>412</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>641</width>
    <height>351</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>738</width>
    <height>412</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>串口助手 1.0</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="CleanSendButton">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>340</y>
     <width>130</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清空发送区域</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SendButton">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>310</y>
     <width>130</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_SendCount">
   <property name="geometry">
    <rect>
     <x>483</x>
     <y>370</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>发送：0</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_ReciveHex">
   <property name="geometry">
    <rect>
     <x>651</x>
     <y>44</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex显示</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CleanReciveButton">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>254</y>
     <width>130</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清空接收区域</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>40</y>
     <width>131</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="Seriallabel">
      <property name="text">
       <string>串  口：</string>
      </property>
      <property name="buddy">
       <cstring>SerialcomboBoxl</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="SerialcomboBoxl"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="Baudlabel">
      <property name="text">
       <string>波特率：</string>
      </property>
      <property name="buddy">
       <cstring>BaudcomboBox</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="BaudcomboBox">
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4800</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="Paritylabel">
      <property name="text">
       <string>校验位：</string>
      </property>
      <property name="buddy">
       <cstring>ParitycomboBox</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="ParitycomboBox">
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Even</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Odd</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Space</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mark</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="Datalabel">
      <property name="text">
       <string>数据位：</string>
      </property>
      <property name="buddy">
       <cstring>DatacomboBox</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="DatacomboBox">
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="Stoplabel">
      <property name="text">
       <string>停止位：</string>
      </property>
      <property name="buddy">
       <cstring>StopcomboBox</cstring>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="StopcomboBox">
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1.5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="controlflowlabel">
      <property name="text">
       <string>控制流：</string>
      </property>
      <property name="buddy">
       <cstring>controlflowcomboBox</cstring>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="controlflowcomboBox">
      <item>
       <property name="text">
        <string>NoFlow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hardward</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Softward</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="ReciveTextEdit">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>44</y>
     <width>461</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(229, 229, 229, 100);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_ShowReciveTime">
   <property name="geometry">
    <rect>
     <x>651</x>
     <y>64</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>时间戳</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_SendLindFeed">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>328</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>自动换行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_emptyCount">
   <property name="geometry">
    <rect>
     <x>568</x>
     <y>364</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清空计数</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_SendHex">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>310</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SwitchButton">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>224</y>
     <width>130</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="SendtextEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_ReceiveCount">
   <property name="geometry">
    <rect>
     <x>391</x>
     <y>370</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>接收：0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_openFile">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sendfile">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送文件</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
